process/process.go

```diff
@@ -19,6 +19,7 @@ func New(name string, nodes []*Process_Node, edges []*Process_Edge) (*Process, e
 	}
 	p.Hash = hash.Dump(p)
 	p.Address = sdk.AccAddress(crypto.AddressHash(p.Hash))
+	p.PaymentAddress = p.Address
```

Comment:
are you sure about this? shouldn't it be set to an address passed by parameter?

Reply:
I wanted to keep the default behavior which is using the `process` address as payment address but that might not be necessary as this is a breaking change anyway.